distribution/docs/src/main/java/ddf/docs/NoOp.java

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Apparently this class is needed to have surefire run the tests in the docs module.